open-metadata-resources/open-metadata-deployment/charts/vdc/values.yaml

```diff
@@ -55,71 +55,61 @@ debug:
 logging:
   level: NONE
 
-
-# The following section defines all of the DOCKER images being used by this
-# chart. Normally they should be left as is, but are exposed here in case the
-# user wishes to extend.
+imageDefaults:
+  registry: "localhost:5000"
+  tag: latest
+  pullPolicy: IfNotPresent
+
+# The following section defines all of the DOCKER images being used by this chart. Normally they should be left as is,
+# but are exposed here in case the user wishes to extend. By default, each will use the 'imageDefaults' above, but
+# these can be individually overridden for each image, if desired, by providing a subkey for 'registry', 'tag' or
+# 'pullPolicy' with the value to use for the override (if you want to use the public Docker registry, use
+# 'docker.io/library' as the registry override).
 image:
   ranger:
-    repository: localhost:5000/odpi/ranger-admin-egeriavdc
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
+    repository: odpi/ranger-admin-egeriavdc
 
   mariadb:
+    registry: "docker.io/library"
     repository: mariadb
-    tag: latest
-    pullPolicy: Always
 
   egeria:
-    repository: localhost:5000/odpi/egeria
-    tag: 1.1-SNAPSHOT
-    pullPolicy: Always
+    repository: odpi/egeria
```

Comment:
the repo name needs to be configurable (see above comment) - clearly it already is, but we can default it too, for the egeria custom images (ie not mariadb)

Reply:
Included in #829